src/dotnet/Users.Service.Migration/Migrations/UsersDbContextModelSnapshot.cs

```diff
@@ -138,6 +138,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .IsRequired()
                         .HasColumnType("text");
 
+                    b.Property<short>("Status")
+                        .HasColumnType("smallint");
+
                     b.Property<long>("Version")
                         .IsConcurrencyToken()
```

Comment:
Definitely prefer attribute for `[Key]` and `[ConcurrencyToken]`.

Overall, there is an agreement that we prefer attributes over model builder stuff, because it's easier to find & read these.

Reply:
I.e. use model builder for whatever is impossible to get w/ attributes.

Reply:
This is just a snapshot file generated with ef. As for db model I of course prefer attribute way.